command/configtest.go

```diff
@@ -0,0 +1,60 @@
+package command
+
+import (
+	"flag"
+	"strings"
+
+	"github.com/hashicorp/consul/command/agent"
+	"github.com/mitchellh/cli"
+)
+
+// ConfigTestCommand is a Command implementation that is used to
+// verify config files
+type ConfigTestCommand struct {
+	Ui cli.Ui
+}
+
+func (c *ConfigTestCommand) Help() string {
+	helpText := `
+Usage: consul configtest [options]
+
+  Test that configs are valid.
```

Comment:
Can we add a more verbose description? It would be helpful to explain what exactly the command does and why it is useful.